addons/controllers/testdata/antrea-test-template-config-1.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: cni.tanzu.vmware.com/v1alpha1
+kind: AntreaConfig
+metadata:
+  name: test-cluster-4
+  namespace: default
```

Comment:
**nit**: Can we make this tkg-system namespace instead of default. Since we expect config-template only in system namespace want to make sure the testdata also reflects the same.

Reply:
Thanks for catching this, will update the namespace.